app/services/idv/steps/threat_metrix_step_helper.rb

```diff
@@ -0,0 +1,52 @@
+module Idv
+  module Steps
+    module ThreatMetrixStepHelper
```

Comment:
@theabrad this might overlap with some of the refactoring / cleanup work you're doing